src/app/components/DeleteInputForm/index.tsx

```diff
@@ -21,7 +21,9 @@ export function DeleteInputForm({ children, onCancel, onConfirm }: DeleteInputFo
       <FormField
         name="type_delete"
         validate={(value: string | undefined) =>
-          !value || value.toLowerCase() !== t('deleteForm.confirmationKeyword', 'delete').toLowerCase()
+          !value ||
+          value.toLowerCase().replace(/['"“”]/g, '') !==
```

Comment:
Doesn't this enable you to write `'''''''''delete'''''''''`?

Reply:
yeah :shrug: still seems fine

Reply:
Also `'''''d'e''l'e''t'e'''''` would be possible. But I guess it doesn't matter in the end.

Reply:
I would personally rather see we remove apostrophes entirely. And mark the text different(italic, bold), that is supposed to be written down in confirmation dialog.

Reply:
good idea
https://github.com/oasisprotocol/oasis-wallet-web/pull/1953